src/view/view.cpp

```diff
@@ -482,6 +482,18 @@ bool wf::view_interface_t::should_be_decorated()
 
 void wf::view_interface_t::set_decoration(surface_interface_t *frame)
 {
+    if (!frame)
+    {
+        damage();
+        if (view_impl->decoration)
+            view_impl->decoration->unref();
+        view_impl->decoration = frame;
```

Comment:
i'd like to see it set to `nullptr` explicitly (frame is null anyway)

Reply:
Yes, makes sense.